res/translations/mixxx_de.ts

```diff
@@ -373,6 +373,11 @@
         <source>Color</source>
         <translation>Farbe</translation>
     </message>
+    <message>
+        <location filename="../../src/library/columncache.cpp" line="132"/>
+        <source>Pitch</source>
+        <translation>Stimmung</translation>
+    </message>
```

Comment:
we keep translation updates separate to update Transifex in one go. Please revert it here.